neo/Network/NeoNode.py

```diff
@@ -86,6 +86,28 @@ def connectionMade(self):
         self.host = self.endpoint.host
         self.port = int(self.endpoint.port)
         self.leader.AddConnectedPeer(self)
+        # write new entry into Banlist (and make sure that node was not already banned)
+        filename = "misbehaving.txt"
+        f = open(filename, 'r+')
+        ip = None
+        for line in f:
+            attributes = line.split(",")
```

Comment:
Does this work without calling f.read()?

Reply:
I never use f.read unless it is implicit in the for line in f loop. Should I avoid it?